src/plugins/inputs/cycle.js

```diff
@@ -0,0 +1,31 @@
+"use strict";
```

Comment:
Mind calling the file `rejoin.js` please? :) I'd like this to be the canonical name, and `/cycle` seen as an alias.